kubectl/kubectl.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# If there is no current context, get one.
+if [[ $(kubectl config current-context 2> /dev/null) == "" ]]; then
+	cluster=$(gcloud config get-value container/cluster 2> /dev/null)
+	zone=$(gcloud config get-value compute/zone 2> /dev/null)
+	function var_usage() {
+		cat <<EOF
+No cluster is set. To set the cluster (and the zone where it is found), set the environment variables
```

Comment:
Does kubectl itself fail with a helpful error message if these values aren't found? It might be better to use the common error path, in case the requirements change in a future release. kubectl's error message will also presumably have more helpful search results, since all anybody ever does is google error messages.